modules/data.atmosphere/NAMESPACE

```diff
@@ -81,6 +85,7 @@ export(wide2long)
 importFrom(PEcAn.DB,db.close)
 importFrom(PEcAn.DB,db.query)
 importFrom(PEcAn.DB,dbfile.input.insert)
+importFrom(magrittr,"%>%")
```

Comment:
For imports like this that are likely to get used in multiple places, I'd prefer we make a habit of putting them in a place that clearly applies to the whole package, e.g. in  PEcAn.DB we put the importFrom in the [package documentation block](https://github.com/PecanProject/pecan/blob/develop/base/db/R/zzz.R#L8). 

That said, maybe it makes sense to import it here first and then move it when it's actually being used multiple places in the package.